### ISSUE TITLE
"arrow-parens": [true, "ban-single-arg-parens"] false alarms at (x): string => 'hey'

### ISSUE THREAD

**Issue by aleksey-bykov** [palantir/tslint#3360]
### Bug Report

- "tslint": "5.0.0-dev.0",
- "typescript": "2.6.0-dev.20171005"
- node ./node_modules/tslint/bin/tslint --project ./scripts/src/tsconfig.json --config ./tslint-standard.json --format msbuild

#### TypeScript code being linted

```ts
// code snippet
const y = (x): string => 'hey';
```

with `tslint.json` configuration:

```json
{
    "rules": {
        "arrow-parens": [ true, "ban-single-arg-parens" ]
    }
}

```

#### Actual behavior
```
Parentheses are prohibited around the parameter in this single parameter arrow function
```
#### Expected behavior
no error

**Comment by ajafff** [comment]
Fixed by #2265 released in 5.0.0
